### ISSUE TITLE
Missing Word

### ISSUE THREAD

**Issue by BartMassey** [stevedonovan/gentle-intro#57]
In 1-basics: "There's a lot of cleverness in the language, but it tries not to anything." I think this is not a clever joke, but rather a missing word or phrase?

**Comment by stevedonovan** [comment]
You are quite right!  It's fixed in master now, but the book needs to be rebuilt - thanks for the heads up.
